packages/react/src/themes/teams/components/Checkbox/checkboxStyles.ts

```diff
@@ -0,0 +1,15 @@
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { CheckboxProps, CheckboxState } from '../../../../components/Checkbox/Checkbox'
+import { CheckboxVariables } from './checkboxVariables'
+import getBorderFocusStyles from '../../getBorderFocusStyles'
+
+const checkboxStyles: ComponentSlotStylesInput<CheckboxProps & CheckboxState, CheckboxVariables> = {
+  root: ({ props: p, variables: v, theme: t }): ICSSInJSStyle => ({
+    ...getBorderFocusStyles({
+      siteVariables: t.siteVariables,
```

Comment:
It seems that no focus styles are visible. The component looks exactly the same when focused:
![image](https://user-images.githubusercontent.com/4512430/58795138-2fb1a400-85fa-11e9-8937-e1d2b9e62f9a.png)

Reply:
Fixed 👍